config/routes.rb

```diff
@@ -7,4 +7,5 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources :sandbox
```

Comment:
it might be nice to keep a sandbox route for development-only purposes, like this
```suggestion
  if Rails.env.development?
    get "/sandbox", to: "sandbox#index"
  end
```